tests/deepsparse/image_classification/test_pipelines.py

```diff
@@ -59,12 +59,13 @@ def test_image_classification_pipeline_preprocessing(
     ic_pipeline = Pipeline.create("image_classification", model_path=zoo_stub)
     zoo_model = Model(zoo_stub)
     data_originals_path = None
-    if zoo_model.sample_originals is not None:
-        if not zoo_model.sample_originals.files:
-            zoo_model.sample_originals.unzip()
-        data_originals_path = zoo_model.sample_originals.path
+    if zoo_model.sample_inputs is not None:
```

Comment:
Do we not have sample_originals anymore?

Reply:
Some models do not have originals.